css/recommendations.css

```diff
@@ -0,0 +1,109 @@
+/* #RecommendationTray {
+  display:grid;
+  grid-template-columns:repeat(auto-fill, minmax(250px, 1fr));
+} */
+
+#RecommendationTray {
+  background-color: white;
+  padding-left: 15px;
+  padding-right: 15px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#RecommendationTray > div{
+  padding:5px;
+  width: 300px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 5px;
+  border-radius: 5px;
+  border: 1px solid black;
+  background-color: #eee;
+  text-align: center;
+}
+
+#RecommendationTray > div > p{
+  width: max-content;
+  display:block;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+#RecommendationTray > div > a{
+  text-align: center;
+}
+a > img{
+  display:block;
+  margin-left:auto;
+  margin-right: auto;
+  margin-top: 4px;
+  align-self: center;
+}
+
+body{
+    max-height: 100%;
+}
+@font-face {
+    font-family: Bebas Neue;
+    src: url("../fonts/BebasNeue.otf");
+}
+@font-face {
+    font-family: Gill Sans Light;
+    src: url("../fonts/GillSans-Light.ttf");
+}
```

Comment:
I see that you repeat some CSS rules 4-5 times in other CSS files like:
https://github.com/internetarchive/wayback-machine-chrome/blob/master/css/similarweb.css
https://github.com/anishsarangi/wayback-machine-chrome/blob/master/css/recommendations.css
etc.
There are many css rules like that.

Isn't it better to have a `common.css` which will be loaded in ALL pages that's needed so that you stop repeating CSS rules?